k8s/namespaces/reform-scan/blob-router-service/aat.yaml

```diff
@@ -67,5 +52,3 @@ spec:
         image: hmctspublic.azurecr.io/reform-scan/blob-router-test:prod-aa64e6c6
```

Comment:
shouldn't be needed afaik it should come from the base file:

https://github.com/hmcts/cnp-flux-config/blob/master/k8s/namespaces/bsp/bulk-scan-processor/bulk-scan-processor.yaml#L21